### PR TITLE
AV-216384:T1LR support for AKO gateway api

### PR DESCRIPTION
**Pull request by akshayhavile**

### CODE REVIEW

File: tests/gatewayapitests/graphlayer/httproute_test.go [hunk 1]
```diff
@@ -2474,3 +2475,91 @@ func TestHttpRouteCreationBeforeGateway(t *testing.T) {
...
+		nodes := aviModel.(*avinodes.AviObjectGraph).GetAviEvhVS()
+		childVS := nodes[0].EvhNodes[0]
+		if childVS.HttpPolicyRefs[0].RequestRules[0].HdrAction[0].Action == nil {
+			return ""
+		}
+		return *childVS.HttpPolicyRefs[0].RequestRules[0].HdrAction[0].Action
+	}, 25*time.Second).Should(gomega.Equal("HTTP_REPLACE_HDR"))
+
+	//reset the field
+	os.Setenv("NSXT_T1_LR", "")
```
Comment (abhishekbsingh) [inline tests/gatewayapitests/graphlayer/httproute_test.go:2560]: nit: can use os.Unsetenv("NSXT_T1_LR")

**Review by abhishekbsingh — APPROVED** [review]
lgtm

**Comment by akshayhavile** [comment]
build ako

**Comment by akshayhavile** [comment]
```sudo docker run \
-e ENDPOINTSLICES_ENABLED="false" \
-w=/go/src/github.com/vmware/load-balancer-and-ingress-services-for-kubernetes \
-v /var/work/load-balancer-and-ingress-services-for-kubernetes:/go/src/github.com/vmware/load-balancer-and-ingress-services-for-kubernetes golang:bullseye \
go test -mod=vendor github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/tests/gatewayapitests/... -failfast -timeout 0 -coverprofile cover-20.out -coverpkg=./...
	github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/tests/gatewayapitests		coverage: 0.0% of statements
ok  	github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/tests/gatewayapitests/graphlayer	460.171s	coverage: 18.5% of statements in ./...
ok  	github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/tests/gatewayapitests/ingestion	220.655s	coverage: 2.8% of statements in ./...
ok  	github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/tests/gatewayapitests/npltests	60.383s	coverage: 15.9% of statements in ./...
ok  	github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/tests/gatewayapitests/status	300.938s	coverage: 12.8% of statements in ./...```

**Pull request merged**